src/components/header/HeaderNav.vue

```diff
@@ -18,15 +18,15 @@
           class="block max-w-0 group-hover:max-w-full transition-all duration-300 h-0.5 bg-dark-purple"
         ></span>
       </HeaderNavItem>
-      <RouterLink
+      <a
         v-if="!userStore.isLoggedIn"
+        href="https://app.adomate.ai/login"
```

Comment:
Should just redirect to app.adomate.ai that way if they're already logged in it will take them to the dashboard. If not it will redirect them to the login page.